docs/flink/flink-getting-started.md

```diff
@@ -547,6 +547,71 @@ RewriteDataFilesActionResult result = Actions.forTable(table)
+| double              | double                     |               |
+| char                | string                     |               |
+| varchar             | string                     |               |
+| string              | string                     |               |
+| binary              | binary                     |               |
+| varbinary           | fixed                      |               |
+| decimal             | decimal                    |               |
+| date                | date                       |               |
+| time                | time                       |               |
+| timestamp           | timestamp without timezone |               |
+| timestamp_ltz       | timestamp with timezone    |               |
+| array               | list                       |               |
+| map                 | map                        |               |
+| multiset            | map                        |               |
+| row                 | struct                     |               |
+| raw                 |                            | Not supported |
+| interval            |                            | Not supported |
+| structured          |                            | Not supported |
+| timestamp with zone |                            | Not supported |
+| distinct            |                            | Not supported |
+| null                |                            | Not supported |
+| symbol              |                            | Not supported |
+| logical             |                            | Not supported |
+
+### Iceberg type to Flink type
+
+This type conversion table describes how Iceberg types are converted to the Flink types. The conversion applies on reading from Iceberg table via Flink.
+
+| Iceberg                    | Flink         | Notes                                            |
+|----------------------------|---------------|--------------------------------------------------|
+| boolean                    | boolean       |                                                  |
+| struct                     | row           |                                                  |
+| list                       | array         |                                                  |
+| map                        | map           |                                                  |
+| integer                    | integer       |                                                  |
+| long                       | bigint        |                                                  |
+| float                      | float         |                                                  |
+| double                     | double        |                                                  |
+| date                       | date          |                                                  |
+| time                       | time          | precision is fixed at 0                          |
```

Comment:
precision is 0? I would expect that to be 6 for microseconds.

Reply:
In flink, the precision of time is not supported and it default precision is 0:
```
in iceberg:
...
      case TIME:
        return new TimeType();
...


in flink:
public static final int DEFAULT_PRECISION = 0;

public TimeType() {
        this(DEFAULT_PRECISION);
    }
```

Of course, this is the behavior of flink, maybe we should not write this precision here.

Reply:
> the precision of time is not supported

Then we should not mention it here.

Reply:
done